src/domain/resource/utils.js

```diff
@@ -651,3 +610,29 @@ export const getTaxPercentage = (resource) => {
 
   return '';
 };
+
+export const getSlotTallness = (resource) => {
+  const [hours, minutes, seconds] = get(resource, 'slot_size', '00:03:00').split(':').map(val => Number(val));
```

Comment:
Should the default value be `00:30:00` instead of `00:03:00`? The method returns correct value nevertheless.

Reply:
You are correct 👍